data/templates/postfix/main.cf

```diff
@@ -131,10 +131,6 @@ smtpd_recipient_restrictions =
     reject_unauth_destination,
     permit
 
-# SRS
-sender_canonical_maps = regexp:/etc/postfix/sender_canonical
```

Comment:
We can't remove this without breaking the forward feature of YunoHost (among other things). The proper fix is to use postsrsd to handle the Sender Rewriting Scheme properly...

Reply:
ok. I agree... I can't test postsrsd right now (I tested but surely not enough). I trust you. Can we keep the header. I think we will need to test the first one... many doubts about it.